Assets/Prefabs/Kleaner.prefab

```diff
@@ -14577,7 +14577,6 @@ GameObject:
   - component: {fileID: 8453875387259748960}
```

Comment:
The `ResourceManager` script has been removed from the Kleaner prefab here in favor of making it its own prefab.